validators/compliance-requirements.adoc

```diff
@@ -0,0 +1,110 @@
+= Validator Node Compliance Requirements
+
+The following document outlines compliance requirements for SKALE Network nodes. Failure to follow these requirements will result in a node forfeiting bounty for each month that it's not in compliance. Please be sure to follow the entire proper setup and backup procedures described in Validator Denali Upgrade documentation to ensure your node is operating as intended.
+
+== Compliance Requirements
+
+The following compliance requirements are required to ensure all nodes meet a standard of performance. 
+
+:warning: Failure to meet ANY ONE of the following Denali compliance requirements will result in a node being marked “Out of Compliance” by the NODE Foundation and the node will be forced to forfeit monthly bounties until it has met compliance requirements.
+
+:note: Compliance requirements may be reviewed and modified at any given time.
```

Comment:
but it doesn't mean they forfeit immediately. If we set this flag 1 day earlier than the get bounty they forfeit but if it happens the first week of the month they have 3 weeks to fix the issue.